search/src/main/java/edu/artic/search/SearchBaseViewModel.kt

```diff
@@ -123,4 +123,8 @@ open class SearchBaseViewModel @Inject constructor(
             }
         }
     }
+
+    open fun onClickSeeAll(header: Header) {
```

Comment:
I think this _could_'ve been an interface or abstract function, but I'm not going to be picky.